testing/test-project-setup.sh

```diff
@@ -53,6 +53,12 @@ https://cloud.google.com/resource-manager/docs/migrating-projects-billing";
   gcloud services enable pubsub.googleapis.com --project $TESTING_PROJECT
   gcloud services enable containerregistry.googleapis.com --project $TESTING_PROJECT
 
+  echo "Giving project permission to cloud-devrel group..."
+  gcloud projects add-iam-policy-binding ${TESTING_PROJECT} \
+    --member="group:cloud-devrel@twosync.google.com" --role="roles/owner"
```

Comment:
We're aiming for this script to be usable by anyone, so while I really appreciate updating this to keep it accurate, I think we should parameterize this similar to folders: if the IAM_OWNER is set, do this, otherwise skip. Let's document the configs we're using for our own flavor on an internal doc.

Reply:
Updated